### DISCUSSION TITLE
🎉 Loco version 0.10.0 was released!

### DISCUSSION THREAD

**Discussion by jondot** [loco-rs/loco#831] [Announcements]
1️⃣ **Refactored Middleware**: Forget manual configs—now you can easily customize, enable, or even remove middleware from the default stack. 🛠️ Simplified, powerful, and flexible! [see the PR](https://github.com/loco-rs/loco/pull/785).

2️⃣ **NEW Background Worker Subsystem Refactor**: We’ve added a Postgres-based worker! Want to go Redis-free? Now you can with Postgres-only support for background jobs [learn more](https://loco.rs/docs/processing/workers/).

3️⃣ **Upgrade & Dev Tweaks**: `validator` upgraded + new developer-experience enhancements. Check the [CHANGELOG](https://github.com/loco-rs/loco/blob/master/CHANGELOG.md).

4️⃣ **Chaos Testing Tool**: Meet `crazy-train` 🚂—a chaos testing tool inspired by [Ozzy Osbourne’s classic](https://www.youtube.com/watch?v=Ui79Uf817YA). It stress-tests the Loco CLI to make it bulletproof!

5️⃣ **Docs & Fixes**: Polished docs, squashed typos, and more!

❤️  Thanks: @jacovdbergh @mikebelanger @lerina

**Comment by adrian5** [comment]
Awesome update! :clap:
